api/db/models/petition.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Enum
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql import func
+from ..base_class import Base
+from .supporter_petitions import supporter_petitions
+import enum
+
+
+class Status(enum.Enum):
+    active = "active"
+    victory = "victory"
+    closed = "closed"
+
+
+class Petition(Base):
+    __tablename__ = "petition"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    image = Column(String)
+    decision_maker = Column(String, nullable=False)
+    location = Column(String, nullable=False)
```

Comment:
What is the location?

Reply:
Simply, for now, it's country, etc in string representation. In the future, I'm going to add localization based on user location and it will expand with an additional table called `locations`, and `location` here will be an FK to it.

Reply:
so location here stands for petition language or country this petition related to?

Reply:
Country this petition related to

Reply:
ok, maybe `region` or `country` then.
Also later you can add a category or tags to improve search experience

Reply:
Ok, sounds reasonable